src/PIL/ImageDraw.py

```diff
@@ -261,24 +261,63 @@ def _multiline_split(self, text):
 
         return text.split(split_character)
 
-    def text(self, xy, text, fill=None, font=None, anchor=None, *args, **kwargs):
+    def text(
+        self,
+        xy,
+        text,
+        fill=None,
+        font=None,
+        anchor=None,
+        strokeWidth=0,
+        strokeFill=None,
+        *args,
+        **kwargs
+    ):
         if self._multiline_check(text):
             return self.multiline_text(xy, text, fill, font, anchor, *args, **kwargs)
```

Comment:
This should pass the `strokeWidth` and `strokeFill` parameters, no?

Reply:
Good point, thanks. I've updated the commit.